src/Qir/Execution/Tools/Executable/QirFullStateExecutable.cs

```diff
@@ -49,7 +49,6 @@ public QirFullStateExecutable(FileInfo executableFile, byte[] qirBytecode, ILogg
 
             LibraryDirectories.Add(new DirectoryInfo(Path.Combine(thisModulePath, "runtimes", osID, "native")));
             LibraryDirectories.Add(new DirectoryInfo(Path.Combine(thisModulePath, "Libraries", osID)));
-            LibraryDirectories.Add(new DirectoryInfo(Path.Combine(thisModulePath, "Libraries")));
```

Comment:
What was the reason for removing this?

Reply:
The dll for the Simulator used to be located directly in the Libraries folder. It has been move in this PR for reasons explained in the other comment thread. Let's consolidate the conversation to be in that thread.